### PR TITLE
Add Indonesian translations

### PR DESCRIPTION
**Pull request by hexatester**
Initial Indonesian translation, more incoming soon. :D

### CODE REVIEW

**Comment by hexatester** [comment]
Let me finish 1-getting-started-lessons first before merging.

**Comment by jlooper** [comment]
wow nice, thanks!

**Comment by hexatester** [comment]
That's enough for now, it's quite long. Ready to merge.

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.id.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+# Pengantar Bahasa Pemrograman dan Alat Keterampilan
+
+Pelajaran ini mencakup dasar-dasar bahasa pemrograman. Topik yang dibahas di sini berlaku untuk sebagian besar bahasa pemrograman modern saat ini. Di bagian 'Alat Keterampilan', Anda akan belajar tentang perangkat lunak yang berguna yang membantu Anda sebagai pengembang.
+
+![Pemrograman Intro](webdev101-programming.png)
+> Catatan sketsa oleh [Tomomi Imura](https://twitter.com/girlie_mac)
+
+## Kuis Pra-Kuliah
+[Kuis pra-kuliah](.github/pre-lecture-quiz.md)
```
Comment (hexatester) [inline 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.id.md:9]: Is it possible to change this into [Kuis pra-kuliah](../.github/pre-lecture-quiz.md)?
Or do we need to create .github folder for translations?

File: 1-getting-started-lessons/2-github-basics/translations/README.id.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+Latihan, latihan, latihan. GitHub memiliki jalur pembelajaran yang bagus yang tersedia melalui [lab.github.com](https://lab.github.com/):
+
+- [Minggu Pertama di GitHub](https://lab.github.com/githubtraining/first-week-on-github)
+
+Anda juga akan menemukan lab yang lebih canggih.
+
+## Tugas
+
+Selesaikan [Minggu Pertama di lab pelatihan GitHub](https://lab.github.com/githubtraining/first-week-on-github)
```
Comment (hexatester) [inline 1-getting-started-lessons/2-github-basics/translations/README.id.md:293]: [Membaca Dokumentasi](assignment.id.md)

**Review by hexatester — COMMENTED** [review]
About links and .github folder.

File: 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.id.md (hunk 1 above)
Comment (hexatester) [reply 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.id.md:9]: What do you think? @jlooper

Comment (hexatester) [reply 1-getting-started-lessons/1-intro-to-programming-languages/translations/README.id.md:9]: I'll add .github folder inside translations folder for now.

**Comment by hexatester** [comment]
Ready to merge! 😄

**Comment by jlooper** [comment]
Thank you! We'd love the full content translated if you can! It's quite a job, grateful for this!

**Pull request merged**